src/gameplaykit.cs

```diff
@@ -82,6 +111,39 @@ interface GKAgent2D {
 		void Update (double seconds);
 	}
 
+	[iOS (10,0), TV (10,0), Mac (10,12, onlyOn64: true)]
+	[BaseType (typeof (GKAgent))]
+	interface GKAgent3D {
+
+		[Export ("position", ArgumentSemantic.Assign)]
+		Vector3 Position {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			set;
+		}
+
+		[Export ("velocity")]
+		Vector3 Velocity {
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+		}
+
+		[Export ("rightHanded")]
+		bool RightHanded { get; set; }
+
+		[Export ("rotation", ArgumentSemantic.Assign)]
+		Matrix3 Rotation { 
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			get;
+			[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+			set;
+		}
+
+		[Export ("updateWithDeltaTime:")]
+		void Update (double seconds);
```

Comment:
agreed, Swift did the same in https://developer.apple.com/library/prerelease/content/releasenotes/General/iOS10APIDiffs/Swift/GameplayKit.html

> Added GKAgent3D.update(deltaTime: TimeInterval)